plugins/auth0/pkg/app/assets/transform_template.tmpl

```diff
@@ -79,19 +85,18 @@
           }
       }
     {{ end }}
-    {{ if $.roles }}, {{ end }}
-    {{ range $i, $element := $.roles }}
-      {{ if $i }},{{ end }}
+  {{ else if eq $object_type "group"}}
       {
-        "id": "{{$element.name}}",
+        "id": "{{.name}}",
         "type": "group",
         "properties":{
           {{ fromEnv "connection_id" "ASERTO_CONNECTION_ID" }}
         }
       }
-    {{ end }}
+  {{ end }}
   ],
   "relations":[
+  {{ if eq $object_type "user" }}
```

Comment:
can groups have nested group members?  if so, would it be better to list the members of each group instead of listing the roles that each user has?

Reply:
auth0 does not support nested roles, but there is an API that list all users associated with a role